scripts/miniapps/LULESH.md

```diff
@@ -0,0 +1,104 @@
+# Instruction for building and running LULESH
+
+Implementation: C++, serial, OpenMP, MPI, or MPI+OpenMP
+
+Useful links:
+- LULESH web site: https://codesign.llnl.gov/lulesh.php
+- Build instructions: https://asc.llnl.gov/CORAL-benchmarks/Summaries/LULESH_Summary_v1.pdf
+
+## How to build
+
+1. Download the most recent "CPU Models" tarball found at the web site
+above.  A top level README mostly describes recent changes and does
+not provide build or run information.
+
```

Comment:
There's a link right above this to some build instructions, but I will point it out again here.